src/platform/packages/shared/response-ops/rule_form/src/common/services/dashboard_service.ts

```diff
@@ -7,55 +7,97 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import type { DashboardStart } from '@kbn/dashboard-plugin/public';
-import type { DashboardAttributes } from '@kbn/dashboard-plugin/server';
+import type { SearchQuery } from '@kbn/content-management-plugin/common';
+import type { ContentManagementPublicStart } from '@kbn/content-management-plugin/public';
+import type { Reference, ContentManagementCrudTypes } from '@kbn/content-management-utils';
+import type { SavedObjectError } from '@kbn/core/public';
+import type { GetIn } from '@kbn/content-management-plugin/common';
+
+const DASHBOARD_CONTENT_TYPE_ID = 'dashboard';
+export type DashboardGetIn = GetIn<typeof DASHBOARD_CONTENT_TYPE_ID>;
+
+export type FindDashboardsByIdResponse = { id: string } & (
+  | { status: 'success'; attributes: any; references: Reference[] }
+  | { status: 'error'; error: SavedObjectError }
+);
 
 export interface DashboardItem {
   id: string;
-  attributes: Pick<DashboardAttributes, 'title'>;
+  attributes: any; // DashboardAttributes is exported in the Dashboard plugin and this causes a cycle dependency. Get feedback on the best approach here
```

Comment:
@nickpeihl DashboardAttributes is exported in the Dashboard plugin. I tried to search for any existing package that exports DashboardAttributes and I found `@kbn/content-packs-schema`

So here's what I tried: 

In this file `x-pack/platform/packages/shared/kbn-content-packs-schema/src/helpers/index.ts` I exported `export type { DashboardAttributes } from '@kbn/dashboard-plugin/common/content_management/v2';` and then I imported like this ` import type { DashboardAttributes } from '@kbn/content-packs-schema';`

Then I ran `yarn test:type_check` and I was getting quite a few ts errors. I didn't push that code on Github, maybe CI would have autocommited some fix for me? 

What do you recommend me to do here?

Reply:
prefer `unknown` over `any`
```suggestion
  attributes: unknown; // DashboardAttributes is exported in the Dashboard plugin and this causes a cycle dependency. Get feedback on the best approach here
```

Reply:
@nickpeihl Shall I copy the types directly here?

```
import type { ControlGroupChainingSystem, ControlLabelPosition } from '@kbn/controls-plugin/common';
import type { RefreshInterval } from '@kbn/data-plugin/common';
interface ControlGroupAttributesV1 {
	chainingSystem?: ControlGroupChainingSystem;
	panelsJSON: string; // stringified version of ControlSerializedState
	ignoreParentSettingsJSON: string;
	controlStyle?: ControlLabelPosition;
}
export interface DashboardAttributes {
	controlGroupInput?: ControlGroupAttributesV1;
	refreshInterval?: RefreshInterval;
	timeRestore: boolean;
	optionsJSON?: string;
	useMargins?: boolean;
	description: string;
	panelsJSON: string;
	timeFrom?: string;
	version?: number;
	timeTo?: string;
	title: string;
	kibanaSavedObjectMeta: {
		searchSourceJSON: string;
	};
}
```